### ISSUE TITLE
Mini Kanban board for notes

### ISSUE THREAD

**Issue by cmeinco** [elnook/notedo#4]
**Summary**
Provide a mechanism to organize notes on the collection view into "swimlanes" or "groups". 

**Intended Outcome**
like github project board but Notedo Notes; turn post its into notes for organizing life.

May need a way to include notes from other collections; but not sure if this is needed.  But probably at the min would want to be able to include notes from the current collection and any sub collections.

**How will it work?**
Should work like adding any other note to a collection; more of a view to organize the notes.    Possibly a smaller post it note which is just the "card" view and not a full note.

![image](https://user-images.githubusercontent.com/1312111/98840390-5c333780-240c-11eb-844f-7d9fca2f1d98.png)

**Comment by davidwashere** [comment]
Trying to think through this, and it might be weird / not make sense if the current collection can be 'viewed' in kanban mode when: 
- not every note from the current collection would be placed
- and notes from other collections could be placed

Would it make sense for 'boards' to live outside of a collection? (at a higher global level for example) - or do you envision having each collection possibly being a different kanban board w/ potential overlap?

**Comment by cmeinco** [comment]
agreed, start out with it being a higher level global type things and see where it goes from there.
